packages/dcos-metrics/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-metrics.git",
-    "ref": "b68c5e128120d0bf96a13aa11000999170a33ed6",
-    "ref_origin": "1.1.0"
+    "ref": "62702c3396deb5b47f4492a5a39d9dfb6f95ab0b",
+    "ref_origin": "master"
```

Comment:
@adam-mesos I created a `1.9.1` tag on dcos-metrics which only has the changes for `DCOS_OSS-925`.   The ref is `a01acda5647c3108cff62851bc694d26d1a503d5`.  I'm not sure if we need to backport any other fixes in dcos-metrics:master into 1.9.1. -- I'll leave that up to @klueska to decide.